lib/eventum/class.mime_helper.php

```diff
@@ -177,13 +177,17 @@ public static function encodeAddress($address)
         if (self::is8bit($address)) {
             // split into name and address section
             preg_match('/(.*)<(.*)>/', $address, $matches);
+
+            $qq = preg_replace_callback(
+                '/([\x80-\xFF]|[\x21-\x2F]|[\xFC]|\[|\])/',
+                function ($m) {
+                    return '=' . strtoupper(dechex(ord(stripslashes($m[1]))));
```

Comment:
and saying "i don't use multibyte characters" is irresponsible, if anyone could send mail in they could be in any encoding; chinese spammers included :)